cypress/e2e/tests/migration/applicationinventory/applications/filter.test.ts

```diff
@@ -117,21 +119,18 @@ describe(["@tier2"], "Application inventory filter validations", function () {
     it("Name filter validations", function () {
         Application.open();
 
-        // Enter an existing name substring and assert
-        var validSearchInput = applicationsList[0].name.substring(0, 11);
-        applySearchFilter(name, validSearchInput);
+        applySearchFilter(name, applicationsList[0].name);
```

Comment:
You shouldn't enter here a full match, instead try to enter first letters only, as it was implemented before

Reply:
@ibragins  in some cases when creating multiple applications theres a chance that up to 11 chars 2 or more apps contain same substring, so the test will fail (it happend locally couple of times), 
anyway raised substring to 15 chars hopefully this wont occur

Reply:
anyway reverted this code to be as it was, just raised the substring to index 15

Reply:
Actually the idea is that it may have several matches. In this case we need to fix that failure instead of this kind of refactoring. Need to change logic a bit.

Reply:
okay as discussed below i shall implement it in a new PR